appveyor.yml

```diff
@@ -55,12 +55,17 @@ for:
       $bypassPackaging = $env:APPVEYOR_PULL_REQUEST_NUMBER -and -not $env:APPVEYOR_PULL_REQUEST_TITLE.Contains("[pack]")
 
       if ($bypassPackaging) {
-        Write-Host "Bypassing artifact publishing for pull request." -ForegroundColor Yellow
+        Write-Host "Bypassing artifact publishing and signing for pull request." -ForegroundColor Yellow
       } else {
         Get-ChildItem buildoutput\*.nupkg | % { Push-AppveyorArtifact $_.FullName -FileName $_.Name -DeploymentName "Binaries" }
 
         Get-ChildItem buildoutput\*.zip | % { Push-AppveyorArtifact $_.FullName -FileName $_.Name -DeploymentName "Runtime" }
       }
-  test_script:

```

Comment:
Keep the tests?

Reply:
Yeah, I will -- I just want to run this through AppVeyor a few times to make sure it works. I don't really want the tests to run (since this has nothing to do with tests), so commenting out to save time.